techniques/ncf_technique/category.xml

```diff
@@ -0,0 +1,6 @@
+<xml>           
+  <name>User Techniques</name>
```

Comment:
This will break if people already have it with a different name IIRC

Reply:
We do not push categories to existing configuration-repository directory

Reply:
won't it be updated by techique-upgrade mechanism?

Reply:
Then the name will be updated like for other categories.

Reply:
the name is not updated when changed in the category file, but a new category is created. This is an old known bug.

Reply:
%µ$£ù#*

Reply:
Now it won't anymore

Reply:
\o/